rkt/images.go

```diff
@@ -268,7 +268,14 @@ func (f *fetcher) fetchImageByName(img string, ascFile *os.File) (string, error)
 			return "", fmt.Errorf("invalid image name for discovery: %s", img)
 		}
 		stderr("rkt: searching for app image %s", img)
-		ep, err := discoverApp(app, f.headers, true)
+
+		var insecure discovery.InsecureOption
+		// FIXME: why do we always accept http?
```

Comment:
Jokes aside, we should add an insecure flag for allowing http connections and use it here. There was even an issue about this, right?